R/simulation-pk-analyses.R

```diff
@@ -0,0 +1,12 @@
+
+#' @title SimulationPKAnalyses
+#' @docType class
+#' @description  PKAnalyses of a simulation (either individual or population simulation)
+SimulationPKAnalyses <- R6Class("SimulationPKAnalyses",
+  inherit = DotNetWrapper,
+  public = list(
+    allPKParametersFor = function(quantityPath) {

```

Comment:
This returns all PKParameter for a given output. We can add more wrapping function as soon as we know how this is going to be used. 

Core exposes a few more method that I could expose here as well, such as
pkParameterFor(quantityPath, parameterName)

To discuss

Reply:
Also here - add R6 namespace reference.

Reply:
I think having `pkParameterFor(quantityPath, parameterName)` is a good idea. Could be implemented as a separate issue or directly here.

Reply:
ok can do